content/tutorials/erc20-paymaster/10.index.md

```diff
@@ -40,19 +40,26 @@ Atlas is a smart contract IDE that lets you write, deploy, and interact with con
 
 ## Setup the Project
 
-1. Initiate a new project by running the command:
+First, initialize a new project by running the command:
 
-    ```sh
-    npx zksync-cli create custom-paymaster-tutorial --template hardhat_solidity
-    ```
+<span id="initialize-hardhat-project" data-test-action></span>
 
-    This creates a new ZKsync Era project called `custom-paymaster-tutorial` with a basic `Greeter` contract.
+```sh
+npx zksync-cli create custom-paymaster-tutorial --template hardhat_solidity
+```
 
-1. Navigate into the project directory:
+This creates a new ZKsync Era project called `custom-paymaster-tutorial` with a basic `Greeter` contract.
 
-    ```sh
-    cd custom-paymaster-tutorial
-    ```
+Next, navigate into the project directory and install the dependencies:
+
+<span id="wait-for-init" data-test-action></span>
+<span id="npm-install" data-test-action></span>
```

Comment:
Could we possibly use a vue component to make adding these test tags easier to write with props, etc? Just a thought, I'm not sure if that affects the test process being a vue component.

Reply:
Yes!